cmake/modules/SearchRootCoreDeps.cmake

```diff
@@ -112,3 +112,5 @@ set(python_include_dirs ${PYTHON_INCLUDE_DIRS} ${OTHER_PYTHON_INCLUDE_DIRS})
 set(python_version_strings ${PYTHON_VERSION_STRING} ${OTHER_PYTHON_VERSION_STRING})
 set(python_under_version_strings ${PYTHON_UNDER_VERSION_STRING} ${OTHER_PYTHON_UNDER_VERSION_STRING})
 set(python_libraries ${PYTHON_LIBRARIES} ${OTHER_PYTHON_LIBRARIES})
+set(numpys_found ${NUMPY_FOUND} ${OTHER_NUMPY_FOUND})
+set(python_version_majors ${PYTHON_VERSION_MAJOR} ${OTHER_PYTHON_VERSION_MAJOR})
```

Comment:
Just some cosmetics: can we define all the python-version related lists one after the other?

We can do: python executables, python include dirs, python libraries, then all the python lists related to versions, then the numpy list the last one.